src/js/tracker.js

```diff
@@ -184,13 +185,22 @@
 			useLocalStorage = argmap.hasOwnProperty('useLocalStorage') ? argmap.useLocalStorage : true,
 
 			// Whether to use cookies
-			configUseCookies = argmap.hasOwnProperty('useCookies') ? argmap.useCookies : true,
+			configUseCookies = argmap.hasOwnProperty('useCookies') ? (
+				helpers.warn(
+					'argmap.useCookies is deprecated. Use argmap.stateStorageStrategy instead.'),
+				argmap.useCookies
+			) : true,
+
+			// Strategy defining how to store the state: cookies, localStorage or none
+			configStateStorageStrategy = argmap.hasOwnProperty('stateStorageStrategy') ?
+				argmap.stateStorageStrategy : (!configUseCookies ? 'none': 'cookie'),
```

Comment:
I chose to not store anything if the user took the time to set `useCookies` to false, what do you guys think?